manifests/kustomize/base/kustomization.yaml

```diff
@@ -9,8 +9,6 @@ bases:
 - metadata
 
 images:
-- name: argoproj/workflow-controller
```

Comment:
There's no usage of argoproj/workflow-controller. So I simply removed this one.

Reply:
I'm pretty certain that the workflow controller is being used.

Reply:
Yes, we are using gcr.io/ml-pipeline/workflow-controller, but not argoproj/workflow-controller.

Reply:
I'm pretty certain it's used.

Reply:
Can you let me know where it is? I couldn't find it when I searched for it.

Reply:
Looks like it's already referencing the gcr image: https://github.com/kubeflow/pipelines/blob/fbb008f593f10c478a565f0765d358e6ae1d121e/manifests/kustomize/base/argo/workflow-controller-deployment.yaml#L39

`argoexec` image is already there.

Reply:
Yes, that's what I found too. Do you have any further concerns?

Reply:
Does that image have the required licenses? Is there some script or cloudbuild entry that can create that image when new Argo version comes out?

Reply:
gcr.io/ml-pipeline/workflow-controller:v2.3.0 doesn't have license

gcr.io/ml-pipeline/workflow-controller:v2.3.0-license-compliance has license

> Is there some script or cloudbuild entry that can create that image when new Argo version comes out?
That's a good point.

I have some dirty scripts for generating the licenses. I made this image from these scripts manually this time.

I think cleaning up these scripts is not a blocker for MKP launch. I will put another PR for it later.